src/Mnemonic.h

```diff
@@ -12,6 +12,10 @@ namespace TW {
 
 /// BIP39 Mnemonic Sentence handling.
 class Mnemonic {
+public:
+    static constexpr size_t MaxWords = 24;
+    static constexpr size_t MinWords = 12;
```

Comment:
I think by the protocol you can do 3 words too?

Reply:
How would it work if you have 24 words + password/passphrase?

Reply:
These limits existed so far, I just made more explicit now.  Valid word counts are 12, 15, 18, 21, and 24.
3 words would be very insecure, easy to brute-force.
Passphrase is additional, used for storage encryption, it is not 25th word.